py/zk/zkns_query.py

```diff
@@ -41,7 +44,7 @@ def _sorted_by_srv_priority(entries):
     priority_map[entry.priority].append(entry)
 
   shuffled_entries = []
-  for priority, priority_entries in sorted(priority_map.iteritems()):
+  for unused_priority, priority_entries in sorted(priority_map.iteritems()):
```

Comment:
So it appears that you are already using your own fork, that's great :-)

So instead of git pull, you should do the following:

# fetch everything from upstream
 git fetch upstream

# merge upstream/master into your fork/branch
# I assume you are using master branch in your fork
# this step syncs your master with upstream/master
git merge upstream/master

# If conflicts happen here, resolve them by hand, why? Just for safety, machine doesn't
# read code

# After done, do this to push to your origin
git push origin

# Then create pull request from your fork, you are done !